arch/x86_64/fentry.S

```diff
@@ -1,8 +1,55 @@
 /* argument passing: %rdi, %rsi, %rdx, %rcx, %r8, %r9 */
 /* return value: %rax */
 /* callee saved: %rbx, %rbp, %rsp, %r12-r15 */
-/* stack frame (with -pg): return addr = 8(%rbp), prev fp = 0(%rbp) */
-/* stack frame (with -fentry): return addr = (%rsp), prev fp = 8(%rsp) */
+/* stack frame (with -pg -mfentry): parent addr = 56(%rsp), child addr = 48(%rsp) */
```

Comment:
The reason I wrote `%rsp` instead of `(%rsp)`  is
the value of `%rsp` is the return address of `__fentry__()`.
(e.g. 0x00000000004006ab)
```
Dump of assembler code for function hello:
                  0x00000000004006a6 <+0>:	callq  0x400550 <__fentry__@plt>
    child addr => 0x00000000004006ab <+5>:	push   %rbp
                  0x00000000004006ac <+6>:	mov    %rsp,%rbp
```
Sorry for my confusion..